src/Microsoft.DotNet.Cli.Utils/ProcessReaper.cs

```diff
@@ -51,7 +51,15 @@ public void NotifyProcessStarted()
         {
             if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
             {
-                _job = AssignProcessToJobObject(_process.Handle);
+                // Limit the use of job objects to versions of Windows that support nested jobs.
+                // This means Windows 8 / Windows Server 2012 or later.
+                // Ideally, we would check for some new API export or OS feature instead of the OS version,
+                // but nested jobs are transparently implemented with respect to the Job Objects API.
+                if (Environment.OSVersion.Version.Major > 6 ||
```

Comment:
Does this API lie on .NET Core like it did on Framework: https://github.com/dotnet/corefx/issues/2602#issuecomment-127827095

Reply:
Yes, but the lie is still sufficient in this case :disappointed_relieved:

Reply:
Worth an additional comment?

Reply:
I'd be happy to add one for clarity.

Reply:
2015 was fun:

"API X is broken"

"Yes, that's why we don't have it on Core."

Reply:
Comment has been updated.